src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));
+    }
```

Comment:
replace 100 with constant 
```suggestion
        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));
    }
```

Reply:
> replace 100 with constant

done